testing/solana-test-validator/run_sdk_tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+### maybe a validator is already running
+pgrep -f solana-test-validator
+if [ $? -eq 0 ]; then
+    echo "solana-test-validator already running"
+    exit 1;
+fi
+
```

Comment:
maybe `set -e` here? then the script will error out if any of the commands error; that's useful if you later want to rely on the result of this script in CI or something

Reply:
Because this script has a lot of background processes (`solana-test-validator` and `solana logs` for each program), wouldn't those continue running if `ts-mocha` had a !=0 exit code?